pkg/kubernetes/Makefile.am

```diff
@@ -5,6 +5,8 @@ kubernetes_DATA = \
 	pkg/kubernetes/cluster.min.html.gz \
 	pkg/kubernetes/kubernetes.min.js.gz \
 	pkg/kubernetes/manifest.json \
+	pkg/kubernetes/deploy.min.js.gz \
+	pkg/kubernetes/nulecule.min.js.gz \
```

Comment:
These two are included in the kubernetes.min.js.gz bundle (see below) and it's not necessary to distribute them separately here.